ff2mpv.js

```diff
@@ -1,94 +1,95 @@
 const contexts = ["link", "image", "video", "audio", "selection", "frame"];
+const CREATE_PROFILE = "createProfile";
+const UPDATE_PROFILE = "updateProfile";
+const DELETE_PROFILE = "deleteProfile";
+const PROFILES = "profiles";
+const OPEN_VIDEO = "openVideo";
+const TITLE = "Play in MPV";
 
 function onError(error) {
   console.log(`${error}`);
 }
 
-const OPEN_VIDEO = 'openVideo';
-let TITLE;
-
-function ff2mpv(url, options = []) {
-  browser.tabs.executeScript({
-    code: "video = document.getElementsByTagName('video');video[0].pause();"
-  });
-  browser.runtime.sendNativeMessage("ff2mpv", { url, options }).catch(onError);
-}
-
-async function getOS() {
-  return browser.runtime.getPlatformInfo().then((i) => i.os);
+function ff2mpv(url, tabId, options = []) {
+  if (tabId != null) {
+    chrome.scripting.executeScript({
```

Comment:
@woodruffw Should we `await` all of these kinds of calls. Currently we just ignore the `Promise`.

Reply:
@eNV25 I think there is no need for that. The block is to try to pause a video in case there is one playing when you call ff2mpv. The promise rejecting or failing doesn't affect and awaiting it will only slow calling the `sendNativeMessage`.